test/test_command_monitoring_spec.py

```diff
@@ -239,5 +241,11 @@ def create_tests():
 
 create_tests()
 
+
+globals().update(generate_test_classes(
+    os.path.join(_TEST_PATH, 'unified'),
+    module=__name__,))
```

Comment:
Please move this to a new file `test/test_command_monitoring_unified.py` and add a corresponding `test/load_balancer/test_command_monitoring_unified.py`.

Reply:
Ah... Thanks for pointing this out. Done.